search/search_quality/assessment_tool/edits.cpp

```diff
@@ -0,0 +1,66 @@
+#include "search/search_quality/assessment_tool/edits.hpp"
+
+#include "base/assert.hpp"
+
+// Edits::RelevanceEditor --------------------------------------------------------------------------
+Edits::RelevanceEditor::RelevanceEditor(Edits & parent, size_t index)
```

Comment:
For readability I'd prefer to keep it there, as the parent is declared first, and index in this parent is declared second.